volume/volume_test.go

```diff
@@ -180,16 +179,15 @@ func TestParseMountRaw(t *testing.T) {
 			`/Program Files (x86)`, // With capitals and brackets
 		},
 		invalid: map[string]string{
-			``:                                   "invalid volume specification: ",
-			`.`:                                  "invalid volume specification: ",
-			`c:`:                                 "invalid volume specification: ",
-			`c:\`:                                "invalid volume specification: ",
-			`../`:                                "invalid volume specification: ",
-			`c:\:../`:                            "invalid volume specification: ",
-			`c:\:/foo:xyzzy`:                     "invalid volume specification: ",
-			`/`:                                  "destination can't be '/'",
-			`/..`:                                "destination can't be '/'",
-			`c:\notexist:/foo`:                   `source path does not exist`,
```

Comment:
Do we have a "positive" test for this? (i.e. this test-case should now pass, correct?)

Reply:
Code wise, there's nothing new to test. The coverage is the same either way. I did change the functional test to validate no error is thrown. Do you still think we should include the positive tests?

Reply:
Well, we previously tested that this should fail, but I couldn't find a test in the Windows `valid` list that now verifies that the test passes with a non-existing path (but I may have overlooked)

Reply:
I've added `valid` tests for Windows and LCOW.